common/models/templates/src/trie-model.ts

```diff
@@ -285,7 +305,7 @@ export default class TrieModel implements LexicalModel {
   }
 
   public traverseFromRoot(): LexiconTraversal {
-    return new Traversal(this._trie['root'], '');
+    return new Traversal(this._trie['root'], '', this._trie.totalWeight);
```

Comment:
```suggestion
    return new Traversal(this._trie.root, '', this._trie.totalWeight);
```

Is this possible? Just weird to see two different ways of referencing member props in one line

Reply:
In the current form:  `this._trie.root` is marked private; using bracket-syntax bypasses this.  Changing that can rectify the issue, so I'll do that.